content/_data/changelogs/weekly.yml

```diff
@@ -8542,6 +8542,64 @@
 
   # pull: 4875 (PR title: Add new line at EOF in config.xml)
 
+- version: '2.251'
+  date: 2020-08-05
+  changes:
+  - type: major bug
```

Comment:
major bug?

This is unlikely to ever happen to a real user, who runs Jenkins without plugins

Reply:
I think that it was assigned "major bug" due to the feedback on the new plugin manager behavior.  See https://github.com/jenkins-infra/jenkins.io/pull/3588#discussion_r464674214

Reply:
This is the text on the installed plugins page that only ever shows up if you have no plugins installed.

I had never seen this text before until I was working on ATH fixes in the weekend and I wanted to see if a plugin was causing a breakage or if the issue was in core...

Reply:
This is the text on the installed plugins page that only ever shows up if you have no plugins installed.

I had never seen this text before until I was working on ATH fixes in the weekend and I wanted to see if a plugin was causing a breakage or if the issue was in core...

Reply:
Got it.  Will correct it.

Reply:
Fixed in d7fc42450d3fd2c3abc0acdb20f01dab73459727